pants_jupyter_plugin/plugin.py

```diff
@@ -381,8 +386,26 @@ def pants_repo(self, pants_repo: str):
       self._display_line(f'ERROR: could not find a valid pants repo at {pants_repo}\n')
       return
 
-    self._display_line(f'Using pants repo at: {pants_repo}\n')
-    self._pants_repo = pants_repo.absolute()
+    # Version check for pants v1 vs v2 flags/behavior.
+    with temporary_dir() as tmp_dir:
+      stderr_path = pathlib.Path(tmp_dir) / 'stderr'
```

Comment:
Is there some subtle issue forcing the stderr hoops? Normally you'd just subprocess.run(...) and consult the CompletedProcess for the stdout, stderr and returncode.